src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -0,0 +1,207 @@
+import { Data, DataSchema } from '../../../../utils/data';
+import { exportCsv } from '../../../../utils/preview';
+import DefaultLayout from '../../../layouts/default';
+import { Button } from '../../../primitives/Button.styles';
+import KeyValueTable, {
+  KeyValueTableType,
+} from '../../../tables/KeyValueTable';
+import StickyHeaderTable from '../../../tables/StickyHeaderTable';
+import { LayoutProps } from '../../types';
+
+const kvSchema: DataSchema = {
+  fields: [
+    { name: 'Title', type: 'varchar' },
+    { name: 'Value', type: 'varchar' },
+  ],
+  pandas_version: '0.0.1', // TODO: Figure out what to set this value to.
+};
+
+const mockMetrics: Data = {
+  schema: kvSchema,
+  data: [
+    ['avg_churn', '0.04'],
+    ['avg_workflows', '455'],
+    ['avg_users', '1.2'],
+    ['avg_users', '5'],
+  ],
+};
+
+const mockChecks: Data = {
+  schema: kvSchema,
+  data: [
+    ['reasonable_churn', 'True'],
+    ['wf_count_small', 'False'],
+    ['bounds_check', 'True'],
+    ['avg_users_check', 'False'],
+    ['warning_check', 'Warning'],
+    ['none_check', 'None'],
+  ],
+};
+
+type ArtifactDetailsHeaderProps = {
+  artifactName: string;
+  createdAt?: string;
+  sourceLocation?: string;
+};
+
+const ArtifactDetailsHeader: React.FC<ArtifactDetailsHeaderProps> = ({
+  artifactName,
+  // TODO: add these back once we have support for getting createdAt and sourceLocation.
+  //createdAt,
+  //sourceLocation,
+}) => {
+  return (
+    <Box width="100%" display="flex" alignItems="center">
+      <FontAwesomeIcon
+        height="24px"
+        width="24px"
+        style={{ marginRight: '8px' }}
+        icon={faCircleCheck}
+        color={'green'}
+      />
+      <Typography variant="h4" component="div">
+        {artifactName}
+      </Typography>
+      {/* <Typography marginTop="4px" variant="caption" component="div">
+                Created: {createdAt}
+            </Typography>
+            <Typography variant="caption" component="div">
+                Source: <Link>{sourceLocation}</Link>
+            </Typography> */}
+    </Box>
+  );
+};
+
+type ArtifactDetailsPageProps = {
+  user: UserProfile;
+  Layout?: React.FC<LayoutProps>;
+};
+
+const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
+  user,
+  Layout = DefaultLayout,
+}) => {
+  const workflow = useSelector((state: RootState) => state.workflowReducer);
+  const navigate = useNavigate();
+  const dispatch: AppDispatch = useDispatch();
+  const { workflowId, workflowDagResultId, artifactId } = useParams();
+  const artifactResult: ArtifactResult | null = useSelector(
+    (state: RootState) => {
+      // First, check if there are any keys in the artifactResults object.
+      const artifactResults = state.workflowReducer.artifactResults;
+      if (Object.keys(artifactResults).length < 1) {
+        return null;
+      }
+
+      return artifactResults[artifactId];
+    }
+  );
+
+  useEffect(() => {
+    document.title = 'Artifact | Aqueduct';
```

Comment:
Add a useEffect here for artifactResult.result and set the name of the webpage to be the artifact's name.

```
  const artifactName: string = artifactResult.result.name;
```

Reply:
✅